### PR TITLE
Bugfixing: earth origins, derelict away, spell range, explosive implant warnings, bears

### PR DESCRIPTION
**Pull request by ggermanicus**
- IPCs can no longer be from Earth.
- The russian derelict's Cyrillic name has been anglicized to prevent parsing errors. космическая actually means space (according to google translate) but naming an away mission Space Station 13 felt bad and 'Kosmonaut' communicates the actual away better.
- All spells with -1 range now have 0, so you can actually cast them.
- Explosive implants will alert common channel when exposed to surgery. The code was already there, it just didn't work. No longer has the 'encased' check since the comments implied the ribcage/skull shouldn't block the tampering alert.
- Bears should now attack normally.
- Parrots do not grab 'the the' item

### CODE REVIEW

**Comment by ErrorsWindows** [comment]
dang

File: code/modules/mob/living/simple_animal/hostile/retaliate/parrot.dm [hunk 1]
```diff
@@ -436,7 +436,7 @@
 				if(!parrot_perch || parrot_interest.loc != parrot_perch.loc)
 					held_item = parrot_interest
 					parrot_interest.forceMove(src)
-					visible_message("[src] grabs the [held_item]!", SPAN_NOTICE("You grab the [held_item]!"), "You hear the sounds of wings flapping furiously.")
+					visible_message("[src] grabs [held_item]!", SPAN_NOTICE("You grab [held_item]!"), "You hear the sounds of wings flapping furiously.")
```
Comment (CPU-Blanc) [inline code/modules/mob/living/simple_animal/hostile/retaliate/parrot.dm:439]: Text macros are your friend
```suggestion
					visible_message("[src] grabs \the [held_item]!", SPAN_NOTICE("You grab \the [held_item]!"), "You hear the sounds of wings flapping furiously.")
```

File: code/modules/mob/living/simple_animal/hostile/retaliate/parrot.dm [hunk 2]
```diff
@@ -603,7 +603,7 @@
 
 			held_item = I
 			I.forceMove(src)
-			visible_message("[src] grabs the [held_item]!", SPAN_NOTICE("You grab the [held_item]!"), "You hear the sounds of wings flapping furiously.")
+			visible_message("[src] grabs [held_item]!", SPAN_NOTICE("You grab [held_item]!"), "You hear the sounds of wings flapping furiously.")
```
Comment (CPU-Blanc) [inline code/modules/mob/living/simple_animal/hostile/retaliate/parrot.dm:606]:
```suggestion
			visible_message("[src] grabs \the [held_item]!", SPAN_NOTICE("You grab \the [held_item]!"), "You hear the sounds of wings flapping furiously.")
```

File: code/modules/mob/living/simple_animal/hostile/retaliate/parrot.dm [hunk 3]
```diff
@@ -631,7 +631,7 @@
 
 		if(stolen_item && C.unEquip(stolen_item, src))
 			held_item = stolen_item
-			visible_message("[src] grabs the [held_item] out of [C]'s hand!", SPAN_WARNING("You snag the [held_item] out of [C]'s hand!"), "You hear the sounds of wings flapping furiously.")
+			visible_message("[src] grabs [held_item] out of [C]'s hand!", SPAN_WARNING("You snag [held_item] out of [C]'s hand!"), "You hear the sounds of wings flapping furiously.")
```
Comment (CPU-Blanc) [inline code/modules/mob/living/simple_animal/hostile/retaliate/parrot.dm:634]:
```suggestion
			visible_message("[src] grabs \the [held_item] out of [C]'s hand!", SPAN_WARNING("You snag \the [held_item] out of [C]'s hand!"), "You hear the sounds of wings flapping furiously.")
```

File: code/modules/mob/living/simple_animal/hostile/retaliate/parrot.dm [hunk 4]
```diff
@@ -666,11 +666,11 @@
 			var/obj/item/grenade/G = held_item
 			G.dropInto(loc)
 			G.detonate()
-			to_chat(src, "You let go of the [held_item]!")
+			to_chat(src, "You let go of [held_item]!")
```
Comment (CPU-Blanc) [inline code/modules/mob/living/simple_animal/hostile/retaliate/parrot.dm:669]:
```suggestion
			to_chat(src, "You let go of \the [held_item]!")
```

File: code/modules/mob/living/simple_animal/hostile/retaliate/parrot.dm [hunk 5]
```diff
@@ -666,11 +666,11 @@
...
 			held_item = null
 			return 1
 
-	to_chat(src, "You drop the [held_item].")
+	to_chat(src, "You drop [held_item].")
```
Comment (CPU-Blanc) [inline code/modules/mob/living/simple_animal/hostile/retaliate/parrot.dm:673]:
```suggestion
	to_chat(src, "You drop \the [held_item].")
```

**Review by CPU-Blanc — COMMENTED** [review]
Text macros _should_ solve these issues better

**Review by CPU-Blanc — APPROVED** [review]

**Comment by Glloyd** [comment]
Squashing this.

**Pull request merged**